app/views/api/v2/hosts/main.json.rabl

```diff
@@ -9,7 +9,12 @@ attributes :ip, :environment_id, :environment_name, :last_report, :mac, :realm_i
            :enabled, :puppet_ca_proxy_id, :managed, :use_image, :image_file, :uuid, :compute_resource_id, :compute_resource_name,
            :compute_profile_id, :compute_profile_name, :capabilities, :provision_method,
            :puppet_proxy_id, :certname, :image_id, :image_name, :created_at, :updated_at,
-           :last_compile, :puppet_status
+           :last_compile, :global_status, :puppet_status
+
+if @host
```

Comment:
When is it false?  Perhaps use rabl's conditionals instead: https://github.com/nesquena/rabl#conditions